spikeinterface/core/mytypes.py

```diff
@@ -0,0 +1,48 @@
+from enum import Enum
+
+class Order(Enum):
+    """Enum for memory order for raw traces in signal block
+    """
+    C = 1  # C order
+    F = 2  # Fortran order
+    K = 3  # Original order
+
+
+class SampleIndex(int):
```

Comment:
@samuelgarcia what would you need this for? 

I would avoid to over engineer the classes and objects used..